### PR TITLE
Travis using conda

### PR DESCRIPTION
**Pull request by FrancescElies**
Installs numpy pandas cython for every environment, mock and unittest2 depending on environment.

### CODE REVIEW

**Comment by esc** [comment]
I didn't know 2.6 was avail through conda, nice.

File: bcolz/tests/test_ctable.py [hunk 1]
```diff
@@ -19,7 +19,10 @@
 from bcolz.py2help import xrange, PY2
 import pickle
 import os
-from mock import Mock
```
Comment (esc) [inline bcolz/tests/test_ctable.py]: Compatability code should live in `bcolz/py2help`.

Comment (FrancescElies) [inline bcolz/tests/test_ctable.py]: I didn't notice we had something for that, I'll move those changes to a new PR, thanks!

**Comment by FrancescElies** [comment]
I would say travis is now ready to use conda

**Comment by FrancescAlted** [comment]
Looks good.  Merging.

**Comment by FrancescAlted** [comment]
Do you have an estimation on how fast this is respect the previous deployment?

**Comment by FrancescAlted** [comment]
Just checked; apparently Travis completes checks 2x faster, which is cool.

**Comment by FrancescElies** [comment]
nice :), appveyor speed up was also quite good, before it took from 6 to 9 minutes for each point in the test-matrix, now it takes between one and two minutes for each point in the test-matrix

**Comment by FrancescAlted** [comment]
Oh, that's much better than I realized.

**Comment by esc** [comment]
Well done!

**Pull request merged**